### PR TITLE
doc/cephadm: add podman version note to install

### PR DESCRIPTION
**Pull request by zdover23**
This PR adds a note to the cephadm installation
guide that informs users that only podman version
2.0.0 and higher work with Ceph, with the
exception of podman version 2.2.1, which does
not work with Ceph. There is also a note regarding
kubic stable 3.0.1 working, but only with newer
kernels.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/install.rst [hunk 1]
```diff
@@ -23,6 +23,10 @@ Requirements
 Any modern Linux distribution should be sufficient.  Dependencies
 are installed automatically by the bootstrap process below.
 
+.. warning::
+
+  Only podman versions that are 2.0.0 and higher work with Ceph Pacific, with the exception of podman version 2.2.1, which does not work with Ceph Pacific. kubic stable is known to work with Ceph Pacific, but it must be run with a newer kernel.
```
Comment (tchaikov) [inline doc/cephadm/install.rst:28]: not sure if we have a table or not. but it'd better to show this as a matrix -- i.e. a table.

Comment (zdover23) [reply doc/cephadm/install.rst:28]: We'll need all the podman z-stream release version numbers in order to create a table.

Comment (liewegas) [reply doc/cephadm/install.rst:28]: Hmm, the thing is that the current/latest ceph works around the podman 2.2.1 problems, so pacific is okay.. it's older octopus version that struggle with 2.2.1... most importantly, they can't upgrade to a later octopus that works (sigh).  I suggest

- we leave off the mention of 2.2.1 here
- we include discussion of 2.2.1 problems (and how to install kubic build) in octopus upgrade notes

Comment (sebastian-philipp) [reply doc/cephadm/install.rst:28]: Can you move this sentence to https://docs.ceph.com/en/latest/cephadm/compatibility/#compatibility-with-podman-versions and just link this section from here?

**Comment by sebastian-philipp** [comment]
ping

File: doc/cephadm/install.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephadm/install.rst:28]: > ping

I'm on it, it's just at the bottom of queue of things.

Comment (sebastian-philipp) [reply doc/cephadm/install.rst:28]: I'm sorry. I'm so behind things.

Comment (zdover23) [reply doc/cephadm/install.rst]: If anyone owes an apology, it's me. (But I don't either.) So let's forgive ourselves.

Anyway: I have made the change. Excelsior!

**Review by sebastian-philipp — APPROVED** [review]

**Pull request merged**